packages/ui5-application-writer/src/data/defaultSettings.ts

```diff
@@ -1,5 +1,6 @@
 import { App, Package, UI5, UI5Framework } from '../types';
```

Comment:
File rename is a temp workaround for sonar/lcov issues reporting the whole file as uncovered.